src/domain/checks/tcSend01.ts

```diff
@@ -7,7 +7,7 @@ export const tcSend01 = Group({
   'Request at least two upload slots': Group({
     'Have you done at least one request?': PreLoadRecordChecks.atLeastOnePreLoadRecordC,
     'Have you required at least one pdf?': PreLoadRecordChecks.atLeastOnePreLoadRecordWithPdfC,
-    'Have you received at least two valid slots?': PreLoadRecordChecks.atLeastOnePreLoadRecordWithPdfC,
+    'Have you received at least two valid slots?': PreLoadRecordChecks.atLeastTwoValidSlotC,
```

Comment:
There is a sort of connection with the activity described by `VL-175`

Reply:
Yep, this can be a great starting point for that activity